src/Features/CSharp/Portable/CodeRefactorings/PullMemberUp/Dialog/CSharpInterfacePullerViaDialog.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.CodeGeneration;
+using Microsoft.CodeAnalysis.CodeRefactorings.PullMemberUp.Dialog;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeRefactorings.PullMemberUp.Dialog
+{
+    [ExportLanguageService(typeof(AbstractInterfacePullerWithDialog), LanguageNames.CSharp), Shared]
+    internal class CSharpInterfacePullerViaDialog : AbstractInterfacePullerWithDialog
```

Comment:
these names don't match.  one is PullerWithDialog, the other is PullerViaDialog.